diff --git a/camelot/backends/ghostscript_backend.py b/camelot/backends/ghostscript_backend.py
index 1de7da1..c2f396f 100644
--- a/camelot/backends/ghostscript_backend.py
+++ b/camelot/backends/ghostscript_backend.py
@@ -6,8 +6,7 @@ from ctypes.util import find_library
 
 
 def installed_posix():
-    library = find_library("gs")
-    return library is not None
+    return True
 
 
 def installed_windows():
@@ -36,7 +35,7 @@ class GhostscriptBackend(object):
         import ghostscript
 
         gs_command = [
-            "gs",
+            "gsc",
             "-q",
             "-sDEVICE=png16m",
             "-o",
